### PR TITLE
[15.0][ADD] web_tree_column_width_remember: Remember the tree view columns

### PR DESCRIPTION
**Pull request by frahikLV**
Remember the tree columns' widths across sessions, and after filtering, grouping, or reordering.

### CODE REVIEW

**Comment by frahikLV** [comment]
@luisg123v review :)

**Comment by luisg123v** [comment]
@carmenmiranda,

Could you leave your :+1: also here, please?

CC @hugho-ad @desdelinux

**Comment by frahikLV** [comment]
@CarmenMiranda, @hugho-ad, @desdelinux
This is a gentle reminder that this PR is pending your review. 

Regards

File: setup/web_remember_tree_column_width/setup.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import setuptools
+
+setuptools.setup(
```
Comment (hugho-ad) [inline setup/web_remember_tree_column_width/setup.py:3]: why this files are need?

**Review by CarmenMiranda — APPROVED** [review]
Technically LGTM 👍🏼

**Comment by hugho-ad** [comment]
@JVegaB could you review pls

File: setup/web_remember_tree_column_width/setup.py (hunk 1 above)
Comment (luisg123v) [reply setup/web_remember_tree_column_width/setup.py:3]: @hugho-ad this is autogenerated by the bot

**Review by hugho-ad — APPROVED** [review]

File: web_remember_tree_column_width/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+* Francisco Javier Luna Vázquez <fluna@vauxoo.com>
+* Tomás Alvárez <tomas@vauxoo.com>
```
Comment (luisg123v) [inline web_remember_tree_column_width/readme/CONTRIBUTORS.rst]:
```suggestion
* Tomás Álvarez <tomas@vauxoo.com>
```

**Review by luisg123v — CHANGES_REQUESTED** [review]

File: web_remember_tree_column_width/readme/CONTRIBUTORS.rst (hunk 2 above)
Comment (frahikLV) [reply web_remember_tree_column_width/readme/CONTRIBUTORS.rst]: @luisg123v Done

**Review by luisg123v — APPROVED** [review]
LGTM :+1: 

Hi @pedrobaeza, @moylop260,

Could you review, please?

CC @desdelinux

**Pull request merged**